java-engine/src/jmh/java/UnleashEngineBenchmark.java

```diff
@@ -0,0 +1,50 @@
+package io.getunleash.engine;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Scope;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.nio.file.Paths;
+import java.io.IOException;
+
+
+@State(Scope.Thread)
+public class UnleashEngineBenchmark {
+
+    private UnleashEngine engine;
+    private final String featureFilePath = "../client-specification/specifications/01-simple-examples.json";
+
+    private static String loadFeaturesFromFile(String filePath) throws IOException {
+        ObjectMapper mapper = new ObjectMapper();
+        JsonNode jsonNode = mapper.readTree(Paths.get(filePath).toFile());
+        JsonNode state = jsonNode.get("state");
+        return state.toString();
+    }
+
+    @Setup
+    public void setUp() {
```

Comment:
try setting the context here and reuse it in the benchmarkFeatureToggle() method

Reply:
Yep! Gave it a spin, no meaningful change to output